polynote-frontend/polynote/ui/util/ui_event.ts

```diff
@@ -102,19 +102,21 @@ export class LoadNotebook extends UIMessage {
     }
 }
 
-export class SelectCell extends UIMessage {
+export class CellSelected extends UIMessage {
     constructor(readonly cell: Cell) { super() }
 
-    static unapply(inst: SelectCell): ConstructorParameters<typeof SelectCell> {
+    static unapply(inst: CellSelected): ConstructorParameters<typeof CellSelected> {
         return [inst.cell]
     }
 }
 
-export class TriggerItem extends UIMessage {
```

Comment:
This is no longer used by the notebook listing?

Reply:
`TriggerItem` seemed redundant as it was doing the same thing as `LoadNotebook`, so I replaced it with `LoadNotebook` for simplicity.